lib/linguist/languages.yml

```diff
@@ -6898,11 +6898,11 @@ TI Program:
   color: "#A0AA87"
   extensions:
   - ".8xp"
-  - ".8xk"
-  - ".8xk.txt"
+  - ".83p"
```

Comment:
Please add a sample for this extension.

Reply:
I've added two samples for commonly made types of programs, a quadratic solver and a number guessing game with different difficulties